main.py

```diff
@@ -1,87 +1,136 @@
-#!/home/harro/meter/env/bin/python3
-# Python script om P1 telegram weer te geven
- 
 import re
 import serial
- 
-# Seriele poort confguratie
-ser = serial.Serial()
- 
-# DSMR 2.2 > 9600 7E1:
-# ser.baudrate = 9600
-# ser.bytesize = serial.SEVENBITS
-# ser.parity = serial.PARITY_EVEN
-# ser.stopbits = serial.STOPBITS_ONE
- 
-# DSMR 4.0/4.2 > 115200 8N1:
-ser.baudrate = 115200
-ser.bytesize = serial.EIGHTBITS
-ser.parity = serial.PARITY_NONE
-ser.stopbits = serial.STOPBITS_ONE
- 
-ser.xonxoff = 0
-ser.rtscts = 0
-ser.timeout = 12
-ser.port = "/dev/ttyUSB0"
-ser.close()
- 
-
-from influxdb import InfluxDBClient
-
+from statistics import mean
 from datetime import datetime
+from typing import List
 
-client = InfluxDBClient()
+import contextlib
+
+
+
+# client = InfluxDBClient()
 
-from statistics import mean
 
-current_wattages = []
-
-def add_wattage(watt_current: float, watt_total_low: float, watt_total_high: float) -> None:
-  global current_wattages
-  current_date = datetime.now().isoformat()
-  current_wattages.append(watt_current)
-  if len(current_wattages) == 10:
-    # done with a minute, lets write
-    avg = mean(current_wattages)
-    json_body = [
-      {
-          "measurement": "energy",
-          "time": current_date,
-          "fields": {
-              "watt_current": float(avg),
-              "watt_total_low": watt_total_low,
-              "watt_total_low": watt_total_high
-          }
-      }
-    ]
-    current_wattages = []
-    client.write_points(json_body)
+# current_wattages = []
 
+# def add_wattage(watt_current: float, watt_total_low: float, watt_total_high: float) -> None:
+#   global current_wattages
+#   current_date = datetime.now().isoformat()
+#   current_wattages.append(watt_current)
+#   if len(current_wattages) == 10:
+#     # done with a minute, lets write
+#     avg = mean(current_wattages)
+#     json_body = [
+#       {
+#           "measurement": "energy",
+#           "time": current_date,
+#           "fields": {
+#               "watt_current": float(avg),
+#               "watt_total_low": watt_total_low,
+#               "watt_total_low": watt_total_high
+#           }
+#       }
+#     ]
+#     current_wattages = []
+#     client.write_points(json_body)
 
 
+class DmsrFourTelegram:
+    current_usage_electricity: float
+    total_usage_electricity_low: float
+    total_usage_electricity_high: float
+
+class DsmrInfo:
```

Comment:
Could be a `namedtuple` if you're not going to add any methods.

Reply:
forgot about namedtuple